### PR TITLE
Lod tutorial documentation

### PR DESCRIPTION
**Pull request by kristynchin-olink**
# Title: Write a very short summary here
**Problem:** Write a description of your problem here.

**Solution:** Write a short summary of your solution.

**Key Features:**

* Key feature 1
* Key feature 2
* ...

## Checklist
- [ ] Make sure you are requesting to **pull a feature/bugfix branch** (right side). This should not be main or develop.
- [ ] Make sure you are make a pull request against either **main or develop** (left side). (Requesting to main should be reserved for bugfixes and new releases)
- [ ] Add or update unit tests (if applicable)
- [ ] Check your code with any unit tests (Run devtools::check() locally)
- [ ] Add neccessary documentation (if applicable)

## Type of changes

What type of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue) (link the issue on the right)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [X ] Documentation Update 
- [ ] Other (explain)

## Further comments
If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, any questions you have, etc...

Consider linking any issues (#issue-number ) or adding @mentions to ensure specific people see it.

### CODE REVIEW

**Review by kathy-nevola — APPROVED** [review]
LGTM

File: OlinkAnalyze/vignettes/LOD.Rmd [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+## Integrating Negative Control LOD
+
+The negative control (NC) LOD method requires at least 10 negative controls in a dataset. Negative control data is available in the standard exported Explore HT and Explore 3072 NPX parquet files. 
+
+A negative control will not contribute to the count of the minimum required NCs if either of the following apply:
+
+ + The negative control contains an assay QC warning across all assays, excluding Olink's internal control assays
+ + The negative control does not pass sample QC criteria (sample QC failure or warning) in all Explore HT blocks or all Explore 3072 panels
+
+For each assay, the median NPX is calculated for all negative controls present in a project. For Explore 3072, overlapping assays are assessed separately, within their respective panels. The LOD for each assay is then calculated as the assay's median negative control NPX value plus 0.2 Standard Deviations.
```
Comment (MasoumehSheikhi) [inline OlinkAnalyze/vignettes/LOD.Rmd:70]: @kathy-nevola 
median negative control NPX value plus 0.2 Standard Deviations.  or 

3 Standard Deviations or 0.2 whichever higher 
?

Comment (kathy-nevola) [reply OlinkAnalyze/vignettes/LOD.Rmd]: Good catch, fixed this

File: OlinkAnalyze/vignettes/LOD.Rmd [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+```
+
+
+## Contact Us
+
+We are always happy to help. Email us with any questions:
+
+-   biostat\@olink.com for statistical services and general stats questions
+
+-   biostattools\@olink.com for Olink Analyze and Shiny app support
```
Comment (kathy-nevola) [inline OlinkAnalyze/vignettes/LOD.Rmd:162]:
```suggestion

```

File: OlinkAnalyze/vignettes/LOD.Rmd [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+In this tutorial, you will learn how to use `olink_lod()` to add LOD information to your Olink Explore dataset. Note that Olink Analyze does not contain example Olink Explore HT or Olink Explore 384/3072 datasets within the package, so external data will be necessary for the code below to work. All file paths should be replaced with a path to your data and fixed LOD reference file (if applicable).
+
+
+## Integrating LOD
+
+Limit of Detection (LOD) is a metric that indicates the lowest measurable value of a protein. LOD can be helpful when performing technical evaluations of NPX datasets, such as calculating CVs. As a note, LOD is less important in downstream statistical analyses as values under LOD typically converge across groups. As such, including data below LOD is unlikely to increase the risk of false positive discoveries. Furthermore, data below LOD can be instrumental in downstream analyses such as biomarker discovery as a protein may be well expressed in one group and not measured in another group. In this case, this protein can be a strong biomarker candidate for specific groups.
```
Comment (kathy-nevola) [inline OlinkAnalyze/vignettes/LOD.Rmd:47]:
```suggestion
Limit of Detection (LOD) is a metric that indicates the lowest measurable value of a protein. LOD can be helpful when performing technical evaluations of NPX™ datasets, such as calculating CVs. As a note, LOD is less important in downstream statistical analyses as values under LOD typically converge across groups. As such, including data below LOD is unlikely to increase the risk of false positive discoveries. Furthermore, data below LOD can be instrumental in downstream analyses such as biomarker discovery as a protein may be well expressed in one group and not measured in another group. In this case, this protein can be a strong biomarker candidate for specific groups.
```

**Pull request merged**